deeppavlov/models/preprocessors/capitalization.py

```diff
@@ -60,3 +63,42 @@ def __call__(self, tokens_batch, **kwargs):
             return zero_pad(cap_batch)
         else:
             return cap_batch
+
+
+def process_word(word, to_lower=False, append_case=None):
+    if all(x.isupper() for x in word) and len(word) > 1:
+        uppercase = "<ALL_UPPER>"
+    elif word[0].isupper():
+        uppercase = "<FIRST_UPPER>"
+    else:
+        uppercase = None
+    if to_lower:
+        word = word.lower()
+    if word.isdigit():
```

Comment:
It is hard to determine what other classes behave just as digits, because 13.25 may be also time which has different morphological contexts compared to usual numbers. I decide not to complicate this too much.